src/Compilers/Test/Utilities/CSharp.Desktop/CSharpTestBase.cs

```diff
@@ -74,6 +75,7 @@ private Action<IModuleSymbol> Translate(Action<ModuleSymbol> action)
             string expectedOutput = null,
             CompilationOptions options = null,
             ParseOptions parseOptions = null,
+            EmitOptions emitOptions = null,
```

Comment:
EmitOptions are added to a longish chain of method calls. Is there a better way to turn on an EmitOption for tests that compile and run instrumented binaries?